From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Paul Gofman <pgofman@codeweavers.com>
Date: Tue, 30 Jul 2024 15:00:09 -0600
Subject: [PATCH] kernelbase: HACK: Force disable direct composition for Banyu
 Lintar Angin - Little Storm.

CW-Bug-Id: #24095
Valve-commit: c7a6ae475a9aebfba2dc0d2e053f29b091dc7737
Picked: clean
Class: valve
Evidence: docs/evidence/2026-09-28-wine-proton-rebase.md
Offered-upstream: no
---
 dlls/kernelbase/process.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/dlls/kernelbase/process.c b/dlls/kernelbase/process.c
index bbb3674..a1bb8e0 100644
--- a/dlls/kernelbase/process.c
+++ b/dlls/kernelbase/process.c
@@ -511,6 +511,7 @@ static const WCHAR *hack_append_command_line( const WCHAR *cmd )
     }
     options[] =
     {
+        {L"Banyu Lintar Angin - Little Storm -.exe", L" --disable_direct_composition=1"},
         {L"Super\\Super.exe", L" --disable_direct_composition=1"},
         {L"A Raven Monologue.exe", L" --use-angle=d3d9"},
         {L"antenna\\antenna.exe", L" --use-angle=d3d9"},
